Libplanet/Blocks/Block.cs

```diff
@@ -391,6 +394,22 @@ internal void Validate(DateTimeOffset currentTime)
                 );
             }
 
+            if (Transactions.Any())
+            {
+                string beforeTxId = Transactions.First().Id.ToString();
```

Comment:
`TxId` seems to implement `IComparable`. Is `ToString()` still needed?

Reply:
Oh, it missed it. I'll update it. :+1: